common/output_stream.cpp

```diff
@@ -1808,6 +1809,8 @@ void SpvReflectToYaml::WriteDescriptorBinding(std::ostream& os, const SpvReflect
   //   uint32_t                            set;
   os << t1 << "set: " << db.set << std::endl;
   //   SpvReflectDescriptorType            descriptor_type;
+  os << t1 << "decorations: " << AsHexString(db.decoration_flags) << " # " << ToStringDecorationFlags(db.decoration_flags)
```

Comment:
**FINAL** nit and happy to merge

this should be `"decoration_flags: "` to match the name of the header field

(idk why we still have these comments, but also adjust the `//   SpvReflectDescriptorType            descriptor_type;` above it)